crates/biome_cli/src/changed.rs

```diff
@@ -24,9 +24,5 @@ pub(crate) fn get_changed_files(
 
     let filtered_changed_files = changed_files.iter().map(OsString::from).collect::<Vec<_>>();
 
-    if filtered_changed_files.is_empty() {
```

Comment:
I believe this check should stay, but we need to add the check if `no_errors_on_unmatched` is `true`

Reply:
I've re-added this check, but now it's done twice - once here, and once in `traverse.rs`